.github/workflows/ci.yml

```diff
@@ -220,7 +220,7 @@ jobs:
         rust-version: [
           # 1.72.0, # current stable
           # beta,
-          nightly-2023-09-10
+          nightly-2023-11-26
```

Comment:
We can actually go back to "stable" and "beta" here now that `1.74` is stable (which includes the `[lints]` feature). Mind changing this back to:

```yaml
	1.74.0 # current stable,
	beta
```